### PR TITLE
feat(build): remove deprecated `es/` build

### PR DESCRIPTION
**Pull request by tmorehouse**
BREAKING CHANGE: removal of deprecated `es/` build directory

### Describe the PR

This PR is for the 2.0.0 stable release, to remove the deprecated `es/` build

To do:

- [x] remove `es/` build commands
- [x] docs update
- [x] remove legacy plugin exports and re-normalize `components/index.js` and `directives/index.js` files
- [x] remove legacy `es` entrypoint index
- [x] remove default exports from _all_ component `index.js` files
- [x] remove default exports from _all_ directive `index.js` files
- [x] move all component/directive/plugin re-exports to `src/index.js`
- [ ] Release stable build

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Enhancement
- [ ] ARIA accessibility
- [ ] Documentation update
- [x] Other (please describe) remove deprecated build

**Does this PR introduce a breaking change?** (check one)

- [ ] No
- [x] Yes (please describe)

Users will no longer be able to import from the `es/` build dirs. Users should be switching to the simplified import methods introduced in `v2.0.0-rc.22`. Legacy plugin names will no longer work as they were only exported from `es/` build.

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples, fix typos`, `chore: fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/components/importdoc.vue [hunk 1]
```diff
@@ -169,7 +166,7 @@ export default {
         return {
           component: this.componentTag(c),
           namedExport: c,
-          importPath: this.componentPath(c)
+          importPath: `bootstrap-vue`
```
Comment (jackmu95) [inline docs/components/importdoc.vue]: We can just use single quotes here.

File: docs/components/importdoc.vue [hunk 2]
```diff
@@ -178,27 +175,15 @@ export default {
         return {
           directive: this.directiveAttr(d),
           namedExport: d,
-          importPath: this.directivePath(d)
+          importPath: `bootstrap-vue`
```
Comment (jackmu95) [inline docs/components/importdoc.vue]: We can just use single quotes here.

Comment (tmorehouse) [reply docs/components/importdoc.vue]: Ah true... so used to typing backticks on something else...

may make this a constant and reference.

Comment (jackmu95) [reply docs/components/importdoc.vue]: Yeah

**Comment by jackmu95** [comment]
@tmorehouse Let's get this done first so we don't have to resolve conflicts all the time.

**Comment by tmorehouse** [comment]
@jackmu95 yeah, that was my thinking as well.

**Pull request merged**